osgp/platform/osgp-adapter-ws-smartmetering/src/main/java/org/opensmartgridplatform/adapter/ws/smartmetering/infra/jms/messageprocessor/DomainResponseMessageProcessor.java

```diff
@@ -18,24 +18,30 @@
 import org.opensmartgridplatform.adapter.ws.schema.smartmetering.notification.NotificationType;
 import org.opensmartgridplatform.adapter.ws.shared.services.NotificationService;
 import org.opensmartgridplatform.adapter.ws.shared.services.ResponseDataService;
-import org.opensmartgridplatform.adapter.ws.smartmetering.application.ApplicationConstants;
 import org.opensmartgridplatform.shared.infra.jms.Constants;
 import org.opensmartgridplatform.shared.infra.jms.CorrelationIds;
 import org.opensmartgridplatform.shared.infra.jms.MessageProcessor;
-import org.opensmartgridplatform.shared.infra.jms.MessageType;
 import org.opensmartgridplatform.shared.infra.jms.ResponseMessageResultType;
-import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
 @Slf4j
 @Component
 public class DomainResponseMessageProcessor implements MessageProcessor {
 
-  /** The message type that a message processor implementation can handle. */
-  protected MessageType messageType;
-
-  @Autowired private NotificationService notificationService;
-  @Autowired private ResponseDataService responseDataService;
+  private final NotificationService notificationService;
+  private final ResponseDataService responseDataService;
+  private final String webserviceNotificationApplicationName;
+
+  public DomainResponseMessageProcessor(
+      final NotificationService notificationService,
+      final ResponseDataService responseDataService,
+      @Value("${web.service.notification.application.name}")
```

Comment:
Consider moving the @Value annotation to a class in the application.config package.

Reply:
Let's discuss this, if values turn in to beans i prefer this

Reply:
See comment concerning the @Value annotations at the `ResendNotificationService` class.